hypothesis-python/src/hypothesis/_settings.py

```diff
@@ -425,36 +384,11 @@ def _max_examples_validator(x):
 """,
 )
 
-settings._define_setting(
-    "buffer_size",
-    default=not_set,
-    validator=lambda x: _ensure_positive_int(x, "buffer_size", since="2019-03-06"),
-    description="The buffer_size setting has been deprecated and no longer does anything.",
-    deprecation_message="The buffer_size setting can safely be removed with no effect.",
-    deprecated_since="2019-07-03",
```

Comment:
This one is *technically* over the line, though I have no objection to removing it. 👍